Traceback (most recent call last):
  File "evaluate_agents.py", line 193, in <module>
    agent.load_weights(root_path=path) 
  File "XXX4/Backend-v2/COMP579-Leaderboard-Backend-New/GROUPS/GROUP_003/agent.py", line 92, in load_weights
    self.model.load_state_dict(torch.load(full_path))
  File "XXX4/anaconda3/envs/leaderboard_env/lib/python3.7/site-packages/torch/serialization.py", line 699, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "XXX4/anaconda3/envs/leaderboard_env/lib/python3.7/site-packages/torch/serialization.py", line 231, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "XXX4/anaconda3/envs/leaderboard_env/lib/python3.7/site-packages/torch/serialization.py", line 212, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './GROUPS/GROUP_003/agent-764-mode1-800000'
